vnext/ReactWindows.sln

```diff
@@ -132,17 +132,20 @@ Global
 		Chakra\Chakra.vcxitems*{c38970c0-5fbf-4d69-90d8-cbac225ae895}*SharedItemsImports = 9
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM = Debug|ARM
```

Comment:
I believe we have been very intentional about leaving the x64 at the top of the list. -- as that makes it the default.  the fact that saving the sln in the UI always reorders them is some very annoying VS behavior.